### PR TITLE
tj 6.0.2 (new formula)

### PR DESCRIPTION
**Pull request by colindean**
- [X] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [X] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

Fixes https://github.com/sgreben/tj/issues/2

### CODE REVIEW

**Comment by colindean** [comment]
Rebased on current master

File: Formula/tj.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+class Tj < Formula
+  desc "Line timestamping tool for stdin"
+  homepage "https://github.com/sgreben/tj"
+  url "https://github.com/sgreben/tj/archive/6.0.2.tar.gz"
+  sha256 "c667f8fb09839a677bd6b3d6c06103bce59ffb2e33ec1bd7b49abfc00f02f5dd"
+  head "https://github.com/sgreben/tj.git"
```
Comment (fxcoudert) [inline Formula/tj.rb]: Remove head

File: Formula/tj.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
+class Tj < Formula
+  desc "Line timestamping tool for stdin"
```
Comment (fxcoudert) [inline Formula/tj.rb]: Remove `for stdin`

File: Formula/tj.rb [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+  def install
+    ENV["GOPATH"] = buildpath
+    (buildpath/"src/github.com/sgreben/tj").install buildpath.children
+    cd "src/github.com/sgreben/tj" do
+      system "make", "binaries/osx_x86_64/tj"
+      bin.install "binaries/osx_x86_64/tj"
+    end
+  end
+  test do
```
Comment (fxcoudert) [inline Formula/tj.rb]: Blank live above `test do`

File: Formula/tj.rb [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    (buildpath/"src/github.com/sgreben/tj").install buildpath.children
+    cd "src/github.com/sgreben/tj" do
+      system "make", "binaries/osx_x86_64/tj"
+      bin.install "binaries/osx_x86_64/tj"
+    end
+  end
```
Comment (fxcoudert) [inline Formula/tj.rb]: Missing `prefix.install_metafiles`

File: Formula/tj.rb [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+  test do
+    pipe_output("#{bin}/tj", "test")
```
Comment (fxcoudert) [inline Formula/tj.rb]: Can we check that the exit code is zero, and match some expected content in the output?

**Review by fxcoudert — DISMISSED** [review]

**Review by ilovezfs — APPROVED** [review]

**Pull request closed without merging**